monai/transforms/io/array.py

```diff
@@ -180,10 +180,10 @@ def __init__(
         for r in SUPPORTED_READERS:  # set predefined readers as default
             try:
                 self.register(SUPPORTED_READERS[r](*args, **kwargs))
-            except OptionalImportError:
-                logging.getLogger(self.__class__.__name__).debug(
-                    f"required package for reader {r} is not installed, or the version doesn't match requirement."
-                )
+            except OptionalImportError as err:
```

Comment:
If the user does not specify the reader, when the first reader package he comes across doesn't exist, he will then report the error directly, which goes against our initial plan.